libs/wingc/src/ast.rs

```diff
@@ -328,6 +330,7 @@ impl Display for UtilityFunctions {
 			UtilityFunctions::Log => write!(f, "log"),
 			UtilityFunctions::Assert => write!(f, "assert"),
 			UtilityFunctions::UnsafeCast => write!(f, "unsafeCast"),
+			UtilityFunctions::Nodeof => write!(f, "nodeof"),
```

Comment:
Not a blocker, but `nodeof` rather than `nodeOf` feels weird since it's not camelcase. I understand why it can kinda feel right when you compare it to something like `typeof` from js, but the distinction there is that `typeof` is an operator whereas `nodeOf` is written as a function.

Reply:
I'm open to either, WDYT @eladb?

Reply:
I prefer `nodeof`.